src/main/java/com/fastmail/jstpierre/petmanager/predicates/Predicates.java

```diff
@@ -0,0 +1,43 @@
+package com.fastmail.jstpierre.petmanager.predicates;
+
+import com.fastmail.jstpierre.petmanager.entities.petowners.PetOwner;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.function.BiPredicate;
+
+public class Predicates {
```

Comment:
J'ajouterais `final` et le constructeur protégé :
```suggestion
public final class Predicates {
    private Predicates() {
        // Constructeur protégé
    }
```